Documentation/dev/developer-guide.md

```diff
@@ -31,7 +31,7 @@ $ pip3 install operator-courier
 
 It is important that you clone the project to the right location in your $GOPATH.
 
-The repository much be located at:
+The repository path should look like this:
```

Comment:
I think that was just a typo... how about just subbing `much` with `must`

Reply:
+1

Reply:
Oh is that what it was. Sure.